CRM/Member/Page/Tab.php

```diff
@@ -261,6 +261,17 @@ public function view() {
    * @return null
    */
   public function edit() {
+    // We're trying to edit existing memberships or create a new one so we'll first check that a membership
+    // type is configured and active, if we don't do this we instead show a permissions error and status bounce.
+    $membershipTypes = \Civi\Api4\MembershipType::get(TRUE)
+      ->addWhere('is_active', '=', TRUE)
+      // we only need one, more is great but a single result lets us proceed!
+      ->setLimit(1)
```

Comment:
```suggestion
      ->setLimit(1)
```
Also unsure of this approach, it feels like the right thing performance wise and it should stack with the condition to return the first result that exists and has `is_active` set to `TRUE`?

Reply:
@eileenmcnaughton I feel like you've provided similar performance-fu on other PRs - do you have an idea whether this is a sane approach?

Reply:
@MikeyMJCO yes - I think it's sane - I just wanted to double check the permissions on membership type were available to everyone with ability to edit a membership (although I suspect we did the same in another PR & have checked that).

This is a low-volume piece of code too so that's worth noting